README.rst

```diff
@@ -1,23 +1,53 @@
-Ex Falso / Quod Libet - A Music Library / Editor / Player
-=========================================================
+Ex Falso & Quod Libet
+=====================
 
-Quod Libet is a music management program. It provides several different
-ways to view your audio library, as well as support for Internet radio and
-audio feeds. It has extremely flexible metadata tag editing and searching
-capabilities.
+.. image:: https://img.shields.io/github/workflow/status/quodlibet/quodlibet/test
+    :target: https://github.com/quodlibet/quodlibet/actions
 
-Ex Falso is a tag editor with the same tag editing interface as Quod Libet.
+.. image:: https://readthedocs.org/projects/quodlibet/badge/?version=latest
+    :target: https://quodlibet.readthedocs.io
 
-https://quodlibet.readthedocs.io
+.. image:: https://img.shields.io/codecov/c/github/quodlibet/quodlibet
+    :target: https://app.codecov.io/gh/quodlibet/quodlibet
 
-.. image:: https://github.com/quodlibet/quodlibet/raw/master/docs/images/paned.png
+.. image:: https://img.shields.io/github/v/release/quodlibet/quodlibet
+    :target: https://github.com/quodlibet/quodlibet/releases
+
+.. image:: https://img.shields.io/github/languages/top/quodlibet/quodlibet
+    :target: https://github.com/quodlibet/quodlibet
+
+.. image:: https://img.shields.io/github/contributors/quodlibet/quodlibet
+    :target: https://github.com/quodlibet/quodlibet/graphs/contributors
+
+An audio library, manager & player
+----------------------------------
+
+`Quod Libet <https://quodlibet.readthedocs.io>`_
+is an audio / music management program.
```

Comment:
this seems to be a bit repetitive (audio management, audio manager). I think it would be better to somehow go into more detail there. Actually that's what partially happens after this

Reply:
OK I'll try another version